lib/ethpy/pyproject.toml

```diff
@@ -42,4 +40,7 @@ lateral = [
 
 [build-system]
 requires = ["flit_core>=3.2"]
-build-backend = "flit_core.buildapi"
\ No newline at end of file
+build-backend = "flit_core.buildapi"
+
+[tool.black]
```

Comment:
black is not an ethpy dependency, which is why we did not specify these tool configs in the individual subpackages. I think you should be able to delete lines 45 & 46.

Reply:
removed